schema/yaml/shared.yml

```diff
@@ -17,6 +17,9 @@ message:
     payload:
       type: object
       description: 'content of message'
+    id:
```

Comment:
This is good. Should we require/suggest this to be a Version 4 UUIDs as defined by RFC 4122?
https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29